pywemo/subscribe.py

```diff
@@ -351,24 +351,33 @@ def _url_resubscribe(
         sid: Optional[str],  # pylint: disable=unsubscriptable-object
         url_fn: SubscribeUrlFn,
     ) -> None:
-        request_headers = headers.copy()
         url = url_fn(device)
+        if sid:
```

Comment:
Can you explain a bit more about why you're unsubscribing here>

Reply:
I added some more details in a [comment below](https://github.com/pavoni/pywemo/pull/236#issuecomment-774325545). And add a link to the comment in the code comments.